<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>UIGraphPrefs</class>
  <widget class="PrefWidget" name="UIGraphPrefs">
    <property name="geometry">
      <rect>
	<x>0</x>
	<y>0</y>
	<width>339</width>
	<height>225</height>
      </rect>
    </property>
    <property name="caption">
      <string>QFrameForm</string>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
	<number>10</number>
      </property>
      <property name="spacing">
	<number>5</number>
      </property>
      <item>
	<widget class="QGroupBox" name="GroupBox7">
	  <property name="frameShape">
	    <enum>QGroupBox::Box</enum>
	  </property>
	  <property name="frameShadow">
	    <enum>QGroupBox::Sunken</enum>
	  </property>
	  <property name="title">
	    <string>Colors</string>
	  </property>
	  <layout class="QGridLayout">
	    <property name="margin">
	      <number>10</number>
	    </property>
	    <property name="spacing">
	      <number>5</number>
	    </property>
	    <item row="1" column="0">
	      <layout class="QHBoxLayout">
		<property name="margin">
		  <number>0</number>
		</property>
		<property name="spacing">
		  <number>5</number>
		</property>
		<item>
		  <widget class="ColorButton" name="ui_dataColor">
		    <property name="minimumSize">
		      <size>
			<width>50</width>
			<height>0</height>
		      </size>
		    </property>
		    <property name="maximumSize">
		      <size>
			<width>50</width>
			<height>32767</height>
		      </size>
		    </property>
		    <property name="whatsThis">
		      <string>Click this button so change the color of the data graph in the recorder.</string>
		    </property>
		  </widget>
		</item>
		<item>
		  <widget class="QLabel" name="label_2">
		    <property name="sizePolicy">
		      <sizepolicy>
			<hsizetype>3</hsizetype>
			<vsizetype>1</vsizetype>
		      </sizepolicy>
		    </property>
		    <property name="text">
		      <string>&amp;Data</string>
		    </property>
		    <property name="buddy">
		      <cstring>ui_dataColor</cstring>
		    </property>
		    <property name="wordWrap">
		      <bool>false</bool>
		    </property>
		  </widget>
		</item>
	      </layout>
	    </item>
	    <item row="0" column="0">
	      <layout class="QHBoxLayout">
		<property name="margin">
		  <number>0</number>
		</property>
		<property name="spacing">
		  <number>5</number>
		</property>
		<item>
		  <widget class="ColorButton" name="ui_bgColor">
		    <property name="minimumSize">
		      <size>
			<width>50</width>
			<height>0</height>
		      </size>
		    </property>
		    <property name="maximumSize">
		      <size>
			<width>50</width>
			<height>32767</height>
		      </size>
		    </property>
		    <property name="whatsThis">
		      <string>Click this button to change the background color of the recorder graph.</string>
		    </property>
		  </widget>
		</item>
		<item>
		  <widget class="QLabel" name="TextLabel2_2_4">
		    <property name="sizePolicy">
		      <sizepolicy>
			<hsizetype>3</hsizetype>
			<vsizetype>1</vsizetype>
		      </sizepolicy>
		    </property>
		    <property name="text">
		      <string>&amp;Background</string>
		    </property>
		    <property name="buddy">
		      <cstring>ui_bgColor</cstring>
		    </property>
		    <property name="wordWrap">
		      <bool>false</bool>
		    </property>
		  </widget>
		</item>
	      </layout>
	    </item>
	    <item row="0" column="1">
	      <layout class="QHBoxLayout">
		<property name="margin">
		  <number>0</number>
		</property>
		<property name="spacing">
		  <number>5</number>
		</property>
		<item>
		  <widget class="ColorButton" name="ui_gridColor">
		    <property name="minimumSize">
		      <size>
			<width>50</width>
			<height>0</height>
		      </size>
		    </property>
		    <property name="maximumSize">
		      <size>
			<width>50</width>
			<height>32767</height>
		      </size>
		    </property>
		    <property name="whatsThis">
		      <string>Click this button to change the color of the grid and grid text in the recorder.</string>
		    </property>
		  </widget>
		</item>
		<item>
		  <widget class="QLabel" name="TextLabel3_2_2">
		    <property name="sizePolicy">
		      <sizepolicy>
			<hsizetype>3</hsizetype>
			<vsizetype>1</vsizetype>
		      </sizepolicy>
		    </property>
		    <property name="text">
		      <string>&amp;Grid</string>
		    </property>
		    <property name="buddy">
		      <cstring>ui_gridColor</cstring>
		    </property>
		    <property name="wordWrap">
		      <bool>false</bool>
		    </property>
		  </widget>
		</item>
	      </layout>
	    </item>
	    <item row="1" column="1">
	      <layout class="QHBoxLayout">
		<property name="margin">
		  <number>0</number>
		</property>
		<property name="spacing">
		  <number>5</number>
		</property>
		<item>
		  <widget class="ColorButton" name="ui_cursorColor">
		    <property name="minimumSize">
		      <size>
			<width>50</width>
			<height>0</height>
		      </size>
		    </property>
		    <property name="maximumSize">
		      <size>
			<width>50</width>
			<height>32767</height>
		      </size>
		    </property>
		    <property name="whatsThis">
		      <string>Click this button to change the color of the cursor at the current sampling position in the recorder.</string>
		    </property>
		  </widget>
		</item>
		<item>
		  <widget class="QLabel" name="label2_4">
		    <property name="sizePolicy">
		      <sizepolicy>
			<hsizetype>3</hsizetype>
			<vsizetype>1</vsizetype>
		      </sizepolicy>
		    </property>
		    <property name="text">
		      <string>Cu&amp;rsor</string>
		    </property>
		    <property name="buddy">
		      <cstring>ui_cursorColor</cstring>
		    </property>
		    <property name="wordWrap">
		      <bool>false</bool>
		    </property>
		  </widget>
		</item>
	      </layout>
	    </item>
	    <item row="2" column="0">
	      <layout class="QHBoxLayout">
		<property name="margin">
		  <number>0</number>
		</property>
		<property name="spacing">
		  <number>5</number>
		</property>
		<item>
		  <widget class="ColorButton" name="ui_startColor">
		    <property name="minimumSize">
		      <size>
			<width>50</width>
			<height>0</height>
		      </size>
		    </property>
		    <property name="maximumSize">
		      <size>
			<width>50</width>
			<height>32767</height>
		      </size>
		    </property>
		    <property name="whatsThis">
		      <string>Click this button to change the color of the cursor at the current sampling position in the recorder.</string>
		    </property>
		  </widget>
		</item>
		<item>
		  <widget class="QLabel" name="label2_2_2">
		    <property name="sizePolicy">
		      <sizepolicy>
			<hsizetype>3</hsizetype>
			<vsizetype>1</vsizetype>
		      </sizepolicy>
		    </property>
		    <property name="text">
		      <string>Start threshold</string>
		    </property>
		    <property name="buddy">
		      <cstring>ui_startColor</cstring>
		    </property>
		    <property name="wordWrap">
		      <bool>false</bool>
		    </property>
		  </widget>
		</item>
	      </layout>
	    </item>
	    <item row="2" column="1">
	      <layout class="QHBoxLayout">
		<property name="margin">
		  <number>0</number>
		</property>
		<property name="spacing">
		  <number>5</number>
		</property>
		<item>
		  <widget class="ColorButton" name="ui_extColor">
		    <property name="minimumSize">
		      <size>
			<width>50</width>
			<height>0</height>
		      </size>
		    </property>
		    <property name="maximumSize">
		      <size>
			<width>50</width>
			<height>32767</height>
		      </size>
		    </property>
		    <property name="whatsThis">
		      <string>Click this button to change the color of the cursor at the current sampling position in the recorder.</string>
		    </property>
		  </widget>
		</item>
		<item>
		  <widget class="QLabel" name="label2_3_2">
		    <property name="sizePolicy">
		      <sizepolicy>
			<hsizetype>3</hsizetype>
			<vsizetype>1</vsizetype>
		      </sizepolicy>
		    </property>
		    <property name="text">
		      <string>E&amp;xternal threshold</string>
		    </property>
		    <property name="buddy">
		      <cstring>ui_extColor</cstring>
		    </property>
		    <property name="wordWrap">
		      <bool>false</bool>
		    </property>
		  </widget>
		</item>
	      </layout>
	    </item>
	  </layout>
	</widget>
      </item>
      <item>
	<widget class="QButtonGroup" name="ButtonGroup11_2">
	  <property name="title">
	    <string>Drawing style</string>
	  </property>
	  <layout class="QGridLayout">
	    <property name="margin">
	      <number>10</number>
	    </property>
	    <property name="spacing">
	      <number>5</number>
	    </property>
	    <item row="1" column="0">
	      <widget class="QLabel" name="TextLabel1_3_3_2">
		<property name="text">
		  <string>Data points:</string>
		</property>
		<property name="alignment">
		  <set>Qt::AlignVCenter|Qt::AlignRight</set>
		</property>
		<property name="wordWrap">
		  <bool>false</bool>
		</property>
	      </widget>
	    </item>
	    <item row="0" column="1">
	      <widget class="QComboBox" name="ui_lineMode">
		<property name="currentIndex">
		  <number>1</number>
		</property>
		<item>
		  <property name="text">
		    <string>No line</string>
		  </property>
		</item>
		<item>
		  <property name="text">
		    <string>Solid line</string>
		  </property>
		</item>
		<item>
		  <property name="text">
		    <string>Dotted line</string>
		  </property>
		</item>
	      </widget>
	    </item>
	    <item row="0" column="0">
	      <widget class="QLabel" name="TextLabel1_6_2">
		<property name="sizePolicy">
		  <sizepolicy>
		    <hsizetype>1</hsizetype>
		    <vsizetype>1</vsizetype>
		  </sizepolicy>
		</property>
		<property name="text">
		  <string>Data:</string>
		</property>
		<property name="alignment">
		  <set>Qt::AlignVCenter|Qt::AlignRight</set>
		</property>
		<property name="wordWrap">
		  <bool>false</bool>
		</property>
	      </widget>
	    </item>
	    <item row="0" column="4">
	      <spacer name="Spacer3_2_2_2">
		<property name="sizeHint">
		  <size>
		    <width>20</width>
		    <height>20</height>
		  </size>
		</property>
		<property name="sizeType">
		  <enum>QSizePolicy::Expanding</enum>
		</property>
		<property name="orientation">
		  <enum>Qt::Horizontal</enum>
		</property>
	      </spacer>
	    </item>
	    <item row="0" column="3">
	      <widget class="QSpinBox" name="ui_lineWidth">
		<property name="maximumSize">
		  <size>
		    <width>50</width>
		    <height>32767</height>
		  </size>
		</property>
		<property name="maximum">
		  <number>5</number>
		</property>
		<property name="minimum">
		  <number>1</number>
		</property>
		<property name="value">
		  <number>2</number>
		</property>
		<property name="whatsThis">
		  <string>Here you can choose the line width of the data graph in the recorder.</string>
		</property>
	      </widget>
	    </item>
	    <item row="0" column="2">
	      <widget class="QLabel" name="TextLabel1">
		<property name="text">
		  <string>Width:</string>
		</property>
		<property name="alignment">
		  <set>Qt::AlignVCenter|Qt::AlignRight</set>
		</property>
		<property name="wordWrap">
		  <bool>false</bool>
		</property>
	      </widget>
	    </item>
	    <item row="1" column="1">
	      <widget class="QComboBox" name="ui_pointMode">
		<item>
		  <property name="text">
		    <string>No points</string>
		  </property>
		</item>
		<item>
		  <property name="text">
		    <string>Circle</string>
		  </property>
		</item>
		<item>
		  <property name="text">
		    <string>Square</string>
		  </property>
		</item>
		<item>
		  <property name="text">
		    <string>Diamond</string>
		  </property>
		</item>
		<item>
		  <property name="text">
		    <string>X</string>
		  </property>
		</item>
		<item>
		  <property name="text">
		    <string>Large Circle</string>
		  </property>
		</item>
		<item>
		  <property name="text">
		    <string>Large Square</string>
		  </property>
		</item>
		<item>
		  <property name="text">
		    <string>Large Diamond</string>
		  </property>
		</item>
		<item>
		  <property name="text">
		    <string>Large X</string>
		  </property>
		</item>
	      </widget>
	    </item>
	    <item row="1" column="2" rowspan="1" colspan="3">
	      <spacer name="Spacer233">
		<property name="sizeHint">
		  <size>
		    <width>20</width>
		    <height>20</height>
		  </size>
		</property>
		<property name="sizeType">
		  <enum>QSizePolicy::Expanding</enum>
		</property>
		<property name="orientation">
		  <enum>Qt::Horizontal</enum>
		</property>
	      </spacer>
	    </item>
	  </layout>
	</widget>
      </item>
      <item>
	<widget class="QCheckBox" name="ui_crosshair">
	  <property name="text">
	    <string>Crosshair cursor</string>
	  </property>
	  <property name="checked">
	    <bool>true</bool>
	  </property>
	</widget>
      </item>
      <item>
	<spacer name="Spacer41">
	  <property name="sizeHint">
	    <size>
	      <width>20</width>
	      <height>20</height>
	    </size>
	  </property>
	  <property name="sizeType">
	    <enum>QSizePolicy::Expanding</enum>
	  </property>
	  <property name="orientation">
	    <enum>Qt::Vertical</enum>
	  </property>
	</spacer>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>ColorButton</class>
      <extends>QWidget</extends>
      <header location="local">colorbutton.h</header>
      <sizehint>
	<width>-1</width>
	<height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
	<hordata>5</hordata>
	<verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
    <customwidget>
      <class>PrefWidget</class>
      <extends>QWidget</extends>
      <header location="global">prefwidget.h</header>
      <sizehint>
	<width>-1</width>
	<height>-1</height>
      </sizehint>
      <container>1</container>
      <sizepolicy>
	<hordata>5</hordata>
	<verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>ui_lineMode</tabstop>
    <tabstop>ui_lineWidth</tabstop>
    <tabstop>ui_pointMode</tabstop>
    <tabstop>ui_crosshair</tabstop>
  </tabstops>
  <images>
    <image name="image0">
      <data format="XPM.GZ" length="646">789c6dd2c10ac2300c00d07bbf2234b7229d1be245fc04c5a3201e4615f430059d0711ff5ddb2e6bb236ec90eed134cb5a19d8ef36602af5ecdbfeeac05dda0798d3abebde87e3faa374d3807fa0d633a52d38d8de6f679fe33fc776e196f53cd010188256a3600a292882096246517815ca99884606e18044a3a40d91824820924265a7923a2e8bcd05f33db1173e002913175f2a6be6d3294871a2d95fa00e8a94ee017b69d339d90df1e77c57ea072ede6758</data>
    </image>
  </images>
</ui>
